### ISSUE TITLE
Error 403 while trying to push hw01 to github

### ISSUE THREAD

**Issue by Najoseph** [cfss-fa20/hw01#53]
So, I originally had no problem updating my document to the server as I followed along with the video, but then I realized I accidentally uploaded the document without adding more detail to my background. I went and redid the whole process, forked the url, cloned it, and then as I tried to push it, I got the error message: 

remote: Permission to cfss-fa20/hw01.git denied to Najoseph.
fatal: unable to access 'https://github.com/cfss-fa20/hw01.git/': The requested URL returned error: 403

I don't know how to fix this. At first I thought maybe it was an authentication error, but I put in purposefully wrong info to verify that it is indeed not an authentication error. Any helpful tips would be greatly appreciated.

**Comment by bensoltoff** [comment]
You [accidentally cloned from the original repository, rather than the fork](https://cfss.uchicago.edu/notes/common-git-problems/#accidentally-cloned-from-the-master-not-the-fork). See the link for instructions on how to reset the URL you are trying to push your commits to.

If that is too confusing at this point or it doesn't work, you can properly fork the repo then clone that to your computer. From there you can edit the files again, and this time you should be able to push successfully

**Comment by Najoseph** [comment]
I think I fixed it? It is showing up now under my original incorrect Pull Request. I realized that I had somehow accidentally deleted the README file and that was causing a whole bunch of problems when I tried to edit the folder and push it back to Git. Once I was able to get the README file back included in my forked url then it was easy. However, when I went to do the final Pull Request it did not give me the option to add a new Pull Request, just add on the the first one I did. I hope that makes sense - and I hope that I actually did this correctly. The good new: now I know the process extremely well for future homework assignments!

**Comment by bensoltoff** [comment]
Yes, you can only have one pull request open at a time. Any additional changes you make after opening it are automatically incorporated. [This is what your repo currently looks like on GitHub](https://github.com/Najoseph/hw01), so that is what we will see when we evaluate your assignment.
